src/sdg/producten/tests/test_update_catalogs.py

```diff
@@ -1,27 +1,286 @@
+            automatisch_catalogus_aanmaken=True,
+            organisatie__owms_end_date=None,
+        )
+
+        self.assertEqual(self.producten_database.count(), 1)
+        call_command("update_catalogs")
+        self.assertEqual(self.producten_database.count(), 2)
+        self.assertEqual(
+            self.producten_versies_database.last().product.referentie_product,
+            ref_product,
+        )
 
     def test_correct_products_if_no_translations(self):
-        pass
+        ref_overheid = LokaleOverheidFactory.create(
+            automatisch_catalogus_aanmaken=False
+        )
+        ref_catalog = ProductenCatalogusFactory.create(
+            lokale_overheid=ref_overheid,
+            is_referentie_catalogus=True,
+        )
+        ref_product = ReferentieProductFactory.create(
+            catalogus=ref_catalog,
+        )
+        ProductVersieFactory.create(
+            product=ref_product,
+            publicatie_datum=date.today(),
+        )
+
+        LokaleOverheidFactory.create(
+            automatisch_catalogus_aanmaken=True,
+            organisatie__owms_end_date=None,
+        )
+
+        self.assertEqual(self.localized_producten_database.count(), 0)
+        call_command("update_catalogs")
+        self.assertEqual(self.localized_producten_database.count(), 2)
+        self.assertEqual(self.localized_producten_database.first().taal, "nl")
+        self.assertEqual(self.localized_producten_database.last().taal, "en")
 
     def test_update_products_with_latest_reference_texts(self):
-        pass
+        ref_overheid = LokaleOverheidFactory.create(
+            automatisch_catalogus_aanmaken=False
+        )
+        ref_catalog = ProductenCatalogusFactory.create(
+            lokale_overheid=ref_overheid,
+            is_referentie_catalogus=True,
+        )
+        ref_product = ReferentieProductFactory.create(
+            catalogus=ref_catalog,
+        )
+        ref_product_versie = ProductVersieFactory.create(
+            product=ref_product,
+            publicatie_datum=date.today(),
+            versie=1,
+        )
+        ref_localized_product_nl = LocalizedProductFactory.create(
+            product_versie=ref_product_versie,
+            taal="nl",
+        )
 
-    def test_update_products_from_active_orgs_only(self):
-        pass
+        LokaleOverheidFactory.create(
+            automatisch_catalogus_aanmaken=True,
+            organisatie__owms_end_date=None,
+        )
+
+        call_command("update_catalogs")
+
+        generated_localized_product = (
+            self.localized_producten_database.filter(taal="nl")
+            .exclude(pk=ref_localized_product_nl.pk)
+            .first()
+        )
+
+        # update referentie product versie and localized product
+        self.producten_versies_database.filter(pk=ref_product_versie.pk).update(
+            versie=2,
+            gewijzigd_op=ref_product_versie.gemaakt_op + datetime.timedelta(days=1),
+        )
```

Comment:
This is NOT how versions work! You create a new ProductVersion object with version number 2...